docker/images/mainnet/Dockerfile

```diff
@@ -54,6 +40,21 @@ COPY docker_launcher.sh /tmp/files/usr/local/bin/docker_launcher.sh
 
 # skycoin image
 FROM $IMAGE_FROM
+ARG BDATE
+ARG SCOMMIT
+
+# Image labels
+LABEL "org.label-schema.name"="Skycoin" \
+      "org.label-schema.description"="Skycoin core docker image" \
+      "org.label-schema.vcs-url"="https://github.com/skycoin/skycoin/tree/develop/docker/images/mainnet" \
+      "org.label-schema.vendor"="Skycoin project" \
+      "org.label-schema.url"="skycoin.net" \
+      "org.label-schema.schema-version"="1.0" \
+      "org.label-schema.build-date"=$BDATE \
+      "org.label-schema.vcs-ref"=$SCOMMIT \
+      "org.label-schema.version"="1.0.0-rc.1" \
```

Comment:
Should be Skycoin version if building from a tag .

Reply:
This is made in #2071